test/exit.test.js

```diff
@@ -37,7 +37,9 @@ test('pino with no args log everything when calling process.exit(0)', async ({ n
   not(actual.match(/world/), null)
 })
 
-test('sync false does not log everything when calling process.exit(0)', async ({ equal }) => {
+const hasWeak = !!global.WeakRef
```

Comment:
nit: I despise double inverses more than coercing checks.

Reply:
eh 🤷 all forms of boolean conversion are a bit shit. `Boolean(WeakRef)` is where you call a class without `new` to cast it. `==` is not good for obvious reasons, `!~~` is too esoteric. I suppose you could use `'WeakRef' in global` instead